src/software/ai/hl/stp/action/kick_action.h

```diff
@@ -18,44 +18,36 @@ class KickAction : public Action
     explicit KickAction();
 
     /**
-     * Returns the next Intent this KickAction wants to run, given the parameters.
+     * Updates the params that can be derived from the world for this action
+     *
+     * @param ball The ball being kicked
+     */
+    void updateWorldParams(const Ball &ball);
+
+    /**
+     * Updates the params that cannot be derived from the world for this action
      *
      * @param robot The robot that should perform the kick
      * @param kick_origin The location where the kick will be taken
      * @param kick_direction The direction the Robot will kick in
```

Comment:
Nit: This should be updated to `kick_target`

Reply:
Good catch! Fixed.